mono/metadata/icall-signatures.h

```diff
@@ -0,0 +1,279 @@
+// mono_icall_sig_int32_ptr_ptr
+// mono_icall_sig_int32_ptr_ptrref
+// mono_icall_sig_long_long_int32
+// mono_icall_sig_long_long_long
+// mono_icall_sig_obj_ptr_int
+// mono_icall_sig_object_int_object
+// mono_icall_sig_object_object_ptr
+// mono_icall_sig_object_ptr_int
+// mono_icall_sig_object_ptr_int32
+// mono_icall_sig_object_ptr_ptr
+// mono_icall_sig_ptr_int32_ptrref
+// mono_icall_sig_ptr_object_ptr
+// mono_icall_sig_ptr_ptr_int
+// mono_icall_sig_ptr_ptr_int32
+// mono_icall_sig_ptr_ptr_ptr
+// mono_icall_sig_ptr_ptr_ptrref
+// mono_icall_sig_ptr_uint32_ptrref
+// mono_icall_sig_uint32_double_double
+// mono_icall_sig_uint32_ptr_int32
+// mono_icall_sig_void_double_ptr
+// mono_icall_sig_void_int32_ptrref
+// mono_icall_sig_void_obj_ptr
+// mono_icall_sig_void_object_object
+// mono_icall_sig_void_object_ptr
+// mono_icall_sig_void_ptr_int
+// mono_icall_sig_void_ptr_int32
+// mono_icall_sig_void_ptr_object
+// mono_icall_sig_void_ptr_ptr
+// mono_icall_sig_void_ptr_ptrref
+// mono_icall_sig_void_uint32_ptrref
+// mono_icall_sig_bool_ptr_int32_ptrref
+// mono_icall_sig_int32_int32_ptr_ptrref
+// mono_icall_sig_int32_ptr_int32_ptr
+// mono_icall_sig_int32_ptr_int32_ptrref
+// mono_icall_sig_object_int_object_object
+// mono_icall_sig_object_object_ptr_ptr
+// mono_icall_sig_object_ptr_int_int
+// mono_icall_sig_object_ptr_int_int32
+// mono_icall_sig_object_ptr_int_ptr
+// mono_icall_sig_object_ptr_ptr_int32
+// mono_icall_sig_ptr_object_int32_int32
+// mono_icall_sig_ptr_object_ptr_ptr
+// mono_icall_sig_ptr_ptr_int_ptr
+// mono_icall_sig_ptr_ptr_int32_ptrref
+// mono_icall_sig_ptr_ptr_ptr_ptr
+// mono_icall_sig_ptr_ptr_ptr_ptrref
+// mono_icall_sig_ptr_ptr_uint32_ptrref
+// mono_icall_sig_void_object_object_ptr
+// mono_icall_sig_void_object_ptr_int32
+// mono_icall_sig_void_ptr_int_object
+// mono_icall_sig_void_ptr_int_ptr
+// mono_icall_sig_void_ptr_object_int32
+// mono_icall_sig_void_ptr_ptr_int
+// mono_icall_sig_void_ptr_ptr_int32
+// mono_icall_sig_void_ptr_ptr_ptr
+// mono_icall_sig_void_ptr_ptr_ptrref
+// mono_icall_sig_int32_object_ptr_ptr_ptr
+// mono_icall_sig_object_object_ptr_ptr_ptr
+// mono_icall_sig_object_ptr_int_int_int
+// mono_icall_sig_ptr_object_int_ptr_ptr
+// mono_icall_sig_ptr_ptr_int32_ptr_ptrref
+// mono_icall_sig_ptr_ptr_ptr_int32_ptrref
+// mono_icall_sig_ptr_ptr_ptr_ptr_ptrref
+// mono_icall_sig_ptr_ptr_ptr_ptrref_ptrref
+// mono_icall_sig_void_object_ptr_int32_int32
+// mono_icall_sig_void_object_ptr_ptr_ptr
+// mono_icall_sig_void_ptr_int_int_object
+// mono_icall_sig_int_int_int_ptr_ptr_ptr
+// mono_icall_sig_int_ptr_int_int_ptr_object
+// mono_icall_sig_object_ptr_int_int_int_int
+// mono_icall_sig_object_ptr_ptr_ptr_ptr_ptr
+// mono_icall_sig_ptr_ptr_int32_ptr_ptr_ptrref
+// mono_icall_sig_void_ptr_ptr_int32_ptr_ptrref
+// mono_icall_sig_int32_ptr_ptr_ptr_ptr_ptr_int32
+// mono_icall_sig_void_ptr_ptr_int32_ptr_ptrref_ptr_ptrref
+
+#define ICALL_SIGS				\
+ICALL_SIG (1, (object))				\
```

Comment:
Depending on desired sort order, put count at the end.
These are sorted, and the expanded list is sorted "the same", but lacks the numbers so it looks messy.
The expanded list is just a comment, so they need not be sorted the same.